src/services/Account/account.js

```diff
@@ -1,17 +1,18 @@
 import { toDeepCase } from '~/utils/apiMapper';
+import { ENV } from '~/utils/environment';
```

Comment:
I am not sure it is a good bane for the service
This service provide some settings data, we already have settings provider here. Why not move it there?